app/views/govuk_publishing_components/components/_input.html.erb

```diff
@@ -92,7 +93,11 @@
     }
   %>
 
-  <% if suffix %>
+  <% if prefix %>
```

Comment:
Prefix and suffix shouldn't be mutually exclusive. For example, income could be '£500/day' requiring the '£' prefix and 'day' suffix. (Of course, we could avoid this scenario with an appropriate question, for example "How much do you earn a day?")